Leanplum/Api/LPVarsApi.h

```diff
@@ -0,0 +1,16 @@
+//
+//  LPSetVarsApi.h
+//  Leanplum
+//
+//  Created by Mayank Sanganeria on 5/21/19.
+//  Copyright © 2019 Leanplum. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface LPVarsApi : NSObject
```

Comment:
are we deleting the `LPSetVarsApi` file?

Reply:
Nope. Its a rename. LPVarsApi will have both get and set apis as per the rest of API classes and point of responsibility.

Reply:
sounds good